olp-cpp-sdk-dataservice-write/test/unit/TestIndexLayerClient.cpp

```diff
@@ -159,6 +158,10 @@ class IndexLayerClientOnlineTest : public IndexLayerClientTestBase {
     return std::make_shared<IndexLayerClient>(
         olp::client::HRN{GetTestCatalog()}, settings);
   }
+
+  std::shared_ptr<olp::http::Network> network_{
```

Comment:
Is there any difference between this and the previous initialization? At the end you still create one and destroy one instance on each test case.
More stable would be to have it in the static SetUpTestSuite() and TearDownTestSuite() and only create it once and destroy it once until we fixed the issues and made it more dynamic